lib/timed.rb

```diff
@@ -30,16 +30,17 @@ def after_steps(steps)
         @duration += Time.now - @my_time
       end
 
-      def after_step_result(keyword, step_match, multiline_arg, status, exception, source_indent, background, *args)
-        super
-        if status == :failed
-          error_msg = "#{exception.message} (#{exception.class.name})\n#{exception.backtrace.join("\n")}"
+      def after_test_step(test_step, result)
+        super(test_step, result)
```

Comment:
Why do you list the arguments explicitely? AFAIK, that without any args, `super` passes the original args to the original implementation of the method. Since there is no change to the args, I would prefer to communicate that by omitting them in the super-call.